pkg/render/fluentd.go

```diff
@@ -273,7 +273,7 @@ func (c *fluentdComponent) daemonset() *appsv1.DaemonSet {
 		},
 		Spec: ElasticsearchPodSpecDecorate(corev1.PodSpec{
 			NodeSelector:                  map[string]string{},
-			Tolerations:                   c.tolerations(),
+			Tolerations:                   tolerateAll(),
```

Comment:
Yeah, I agree, I've restored it back to just the two effects. 

I was assuming that since all our other components tolerate those two plus CriticalAddonsOnly, it's safe to do the same here, but lets keep functionality the same in this PR :+1: